src/guide/api.md

```diff
@@ -366,21 +366,52 @@ HTTP API errors will have the following response body:
+- `urlTemplate`: This is the URL that users will be directed to when they click the link. It should be a valid URL except for the token template string, `<token>`. We will swap `<token>` with the actual token value before sending the email. In your application, you should parse the token out of the url (most easily done with a query parameter as seen below) and send it to the `signin/verify` endpoint to validate the request.
+- `userId`: The identifier of the user the email is intended for.
+
+```http request
+POST https://v4.passwwordless.dev/magic-links/send HTTP/1.1
+ApiSecret: myapplication:secret:11f8dd7733744f2596f2a28544b5fbc4
+Content-Type: application/json
+
+{
+  "emailAddress": "user-email@example.com",
+  "urlTemplate": "https://www.myapp.com?token=<token>"
+  "userId": "c8a32e5b-46d3-4808-ae10-16d3e26ff6f9"
+}
+```
+
+### Response
+
+If successful, the `/magic-links/send` endpoint will return an HTTP 204 (No Content) [status code](#status-codes).
+
+If Magic Links has not been enabled, the `/magic-links/send` endpoint will return an HTTP 403 (Unauthorized) [status code](#status-codes) along with a message about enabling the Magic Links feature.
+
 ### Status codes
 
 The API returns HTTP Status codes for each request.
 
 In case you receive an error, you will also receive a JSON serialized summary of the error in the form of [problem details](errors/#problem-details). For more information, see the [Errors page](errors.md).
 
-| HTTP Code | Message                                | Status |
-| --------- | -------------------------------------- | ------ |
-| 200       | Everything is OK.                      | ✅     |
-| 201       | Everything is OK, resource created.    | ✅     |
-| 204       | Everything is OK, response is empty.   | ✅     |
-| 400       | Bad request.                           | 🔴     |
-| 401       | You did not identify yourself.         | 🔴     |
-| 409       | Conflict (alias is already in use).    | 🔴     |
-| 500       | Something went very wrong on our side. | 🔴     |
+| HTTP Code | Message                                                                         | Status |
+| --------- | ------------------------------------------------------------------------------- | ------ |
+| 200       | Everything is OK.                                                               | ✅     |
+| 201       | Everything is OK, resource created.                                             | ✅     |
+| 204       | Everything is OK, response is empty.                                            | ✅     |
+| 400       | Bad request. (see problem details for more info).                               | 🔴     |
+| 401       | You did not identify yourself.                                                  | 🔴     |
+| 403       | You are not allowed to perform the aciton. (see problem details for more info). | 🔴     |
+| 409       | Conflict (see problem details for more info).                                   | 🔴     |
```

Comment:
Can you also please add `429` for being rate-limited (or exceeding monthly quota, in case of magic links)?